src/controllers/request-controller.ts

```diff
@@ -135,6 +135,7 @@ export class RequestController {
         )
       }
       logger.debug(`Found request for ${requestParams.cid} of stream ${requestParams.streamId}`)
+      Metrics.count(METRIC_NAMES.ANCHOR_REQUEST_ACCEPTED, 1, { source: parseOrigin(req) })
```

Comment:
I think it will be helpful to rename these metrics so that they accurately represent what's happening in the API. Let me know your thoughts @gvelez17 :) 
My suggestions for the metrics names: 
METRIC_NAMES.REQUEST_INVALID ---rename_to--> INVALID_REQUEST_COUNT ( validation failure) 
METRIC_NAMES.ANCHOR_REQUESTED ---rename_to--> NEW_ANCHOR_REQUESTED_COUNT ( this will be a new anchor request unique per model creation / model updation )
METRIC_NAMES.ANCHOR_REQUEST_ACCEPTED ---rename_to--> EXISTING_ANCHOR_RETRIEVED_COUNT ( duplicate anchor request, cas already knows about this request )
METRIC_NAMES.REQUEST_NOT_CREATED ---rename_to--> ANCHOR_CREATION_FAILED_COUNT (Something Went Wrong)

Reply:
ok let me review those separately, since other folks could possibly be reading those or already have them in charts.  I think that can be separate from adding the desired metrics here?

Reply:
@samika98 i created a separate ticket for the renaming, i don't want to slow down adding the metrics but i want to give people a chance to check if the renaming will affect existing charts, so i put a ticket here: https://linear.app/3boxlabs/issue/WS1-1628/clarify-cas-metric-names-and-meanings

Reply:
A separate PR sounds good!